Dockerfile

```diff
@@ -1,5 +1,34 @@
+# The CI image needs to build Go and Java tests and run Python tests,
+# so it must contain the bulding tools.
 FROM sqlflow:dev
 
+# Install dependencies.
+COPY docker/ci/js /ci/js
+# Required by install-jupyter.bash
+ENV IPYTHON_STARTUP /root/.ipython/profile_default/startup/
+
+RUN apt-get -qq update
+
+COPY docker/ci/install-pips.bash /ci/
+RUN /ci/install-pips.bash
+
+COPY docker/ci/install-jupyter.bash /ci/
+RUN /ci/install-jupyter.bash
+
+COPY docker/ci/install-mysql.bash /ci/
+RUN /ci/install-mysql.bash
+
+COPY docker/ci/install-odps.bash /ci/
+RUN /ci/install-odps.bash
+
+ENV HADOOP_VERSION 3.2.1
+ENV PATH /opt/hadoop-${HADOOP_VERSION}/bin:/usr/local/go/bin:/go/bin:$PATH
+COPY docker/ci/install-hadoop.bash /ci/
+RUN /ci/install-hadoop.bash
+
+COPY docker/ci/install-kubectl.bash /ci/
+RUN /ci/install-kubectl.bash
```

Comment:
I added these COPY and RUN pairs to make it quick to correct errors while developing this Dockerfile.  In a future PR, we can merge them into one pair to reduce the number of Docker layers.

```bash
COPY docker/ci /ci
RUN /ci/install-pips.sh && /ci/install-jupyter.sh && ...
```